modules/apps/object/object-item-selector-web/src/main/java/com/liferay/object/item/selector/web/internal/ObjectDefinitionTableItemView.java

```diff
@@ -28,7 +28,7 @@ public ObjectDefinitionTableItemView(ObjectDefinition objectDefinition) {
 
 	@Override
 	public List<String> getHeaderNames() {
-		return ListUtil.fromArray("label", "modified-date");
+		return ListUtil.fromArray("label", "scope", "modified-date");
```

Comment:
@jonathanmccann is it possible to sort alphabetically here? `modified-date` before `scope`